enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionResult/InteractiveQuestionResult.tsx

```diff
@@ -45,7 +45,9 @@ const ContentView = ({
     .with("editor", () => (
       <InteractiveQuestion.Editor onApply={onReturnToVisualization} />
     ))
-    .otherwise(() => <InteractiveQuestion.QuestionVisualization />);
+    .otherwise(() => (
+      <InteractiveQuestion.QuestionVisualization height="100%" />
```

Comment:
I guess this is because the default view already has a predetermined height, so we can make the viz full-height?